### PR TITLE
contrib tweaks

### PR DESCRIPTION
**Pull request by spaette**
1. replace the .svg file
2. add a smaller scaled 48x48 .png suitable for KDE's Application Launcher (e.g. for 1366x768 Laptop Screens)
3. aside from updating contrib/README.md based on the foregoing, add content regarding gtk-update-icon-cache
4. comment the .desktop file, regarding the use of alternate terminals (than system defaults) and the use of a .png icon
5. fixed .appdata.xml  \<image\> element url

### CODE REVIEW

**Comment by rfourquet** [comment]
It would be great to give a better title to this PR, and describe what it does in he OP.

**Comment by spaette** [comment]
> devs can change the initial commit message and/or squash

@rfourquet

You're welcome to change the PR title and to squash and use a different commit message.

Maybe something like "contrib (UI improvements)" as a new title?

The OP has now been edited. The linked to Issue was a WIP over a long time. After the Draft PR was opened clarification was received regarding the .svg and the correct .appdata.xml file \<image\> url was located.

`https://github.com/JuliaLang/julia/issues/26947`

Regarding UI see also the last two comments on the above Issue.

I suppose some software might eventually make use of a text/julia mime type.

**Comment by ViralBShah** [comment]
Are these changes causing the tests to fail? Hard to imagine, but I have hit Update.

File: contrib/julia.appdata.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
  </description>
  <screenshots>
   <screenshot type="default">
-   <image>https://julialang.org/images/julia-gnome.png</image>
+   <image>https://raw.githubusercontent.com/JuliaLang/www.julialang.org/main/_assets/images/julia-gnome.png</image>
```
Comment (vchuravy) [inline contrib/julia.appdata.xml:31]: We should not point to GitHub, but to our own CDN

Comment (spaette) [reply contrib/julia.appdata.xml:31]: > point to our own CDN

Noted.

It was the only same named file able to be located.

https://github.com/JuliaLang/www.julialang.org/pull/1773

Comment (ViralBShah) [reply contrib/julia.appdata.xml:31]: Sorry, I didn't realize this PR also needed updating with the new URL. Can we do that in a follow-up?

Comment (spaette) [reply contrib/julia.appdata.xml:31]: anyone can fix

this looks like the CDN url

https://julialang.org/assets/images/julia-gnome.png

contrib/julia.appdata.xml

```diff
    <screenshot type="default">
-    <image>https://raw.githubusercontent.com/JuliaLang/www.julialang.org/main/_assets/images/julia-gnome.png</image>
+    <image>https://julialang.org/assets/images/julia-gnome.png</image>
    </screenshot>
```

Comment (ViralBShah) [reply contrib/julia.appdata.xml:31]: Fixed in #47471

**Pull request merged**